toolchest/pyomoio.py

```diff
@@ -0,0 +1,217 @@
+"""pyomoio: read data from Pyomo models to pandas DataFrames
+
+Pyomo is a AMPL-like model description language for mathematical
+optimization problems. This module provides functions to read data from 
+Pyomo model instances and result objects. Use list_entities to get a list
+of all entities (sets, params, variables, objectives or constraints) inside a 
+pyomo instance, before get its contents by get_entity (or get_entities).
+    
+"""
+import pyomo.core as pyomo
+import pandas as pd
+
+def get_entity(instance, name):
+    """Return a DataFrame for an entity in model instance
+    
+    Parameters
+    ----------
+    instance: a Pyomo ConcreteModel instance
+    name: str, name of a Set, Param, Var, Constraint or Objective in instance
```

Comment:
from https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt, type and description should be on separate lines

```
Parameters
----------
x : type
    Description of parameter `x`.
y
    Description of parameter `y` (with type not specified)
```